<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
	<property name="geometry">
	 <rect>
		<x>0</x>
		<y>0</y>
		<width>793</width>
		<height>402</height>
	 </rect>
	</property>
	<property name="windowTitle">
	 <string>Pchat Client</string>
	</property>
	<widget class="QWidget" name="centralwidget">
	 <layout class="QGridLayout" name="gridLayout">
		<item row="1" column="0">
		 <widget class="QPushButton" name="pushButton_2">
			<property name="sizePolicy">
			 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
				<horstretch>0</horstretch>
				<verstretch>0</verstretch>
			 </sizepolicy>
			</property>
			<property name="text">
			 <string>close</string>
			</property>
		 </widget>
		</item>
		<item row="1" column="3">
		 <widget class="QPushButton" name="pushButton">
			<property name="text">
			 <string>Send</string>
			</property>
		 </widget>
		</item>
		<item row="1" column="1">
		 <widget class="QLabel" name="label">
			<property name="text">
			 <string>Message :</string>
			</property>
		 </widget>
		</item>
		<item row="1" column="2">
		 <widget class="QLineEdit" name="lineEdit"/>
		</item>
		<item row="0" column="0">
		 <widget class="QTreeWidget" name="treeWidget">
			<property name="sizePolicy">
			 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
				<horstretch>0</horstretch>
				<verstretch>0</verstretch>
			 </sizepolicy>
			</property>
			<column>
			 <property name="text">
				<string>Rooms</string>
			 </property>
			</column>
		 </widget>
		</item>
		<item row="0" column="1" colspan="3">
		 <widget class="QTextEdit" name="textEdit">
			<property name="sizePolicy">
			 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
				<horstretch>0</horstretch>
				<verstretch>0</verstretch>
			 </sizepolicy>
			</property>
		 </widget>
		</item>
	 </layout>
	</widget>
	<widget class="QMenuBar" name="menuBar">
	 <property name="geometry">
		<rect>
		 <x>0</x>
		 <y>0</y>
		 <width>793</width>
		 <height>29</height>
		</rect>
	 </property>
	 <widget class="QMenu" name="menuMenu">
		<property name="title">
		 <string>Menu</string>
		</property>
		<addaction name="separator"/>
		<addaction name="actionHome"/>
		<addaction name="actionExit"/>
	 </widget>
	 <addaction name="menuMenu"/>
	</widget>
	<action name="actionHome">
	 <property name="text">
		<string>Home</string>
	 </property>
	</action>
	<action name="actionExit">
	 <property name="text">
		<string>Exit</string>
	 </property>
	</action>
 </widget>
 <resources/>
 <connections/>
</ui>
